src/components/ThoughtsForm.jsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from "react";
+
+const APIURL = "https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts"
+
+export const ThoughtsForm = () => {
+  const [newThought, setNewThought] = useState('')
+  const [error, setError] = useState("")
+
+  const handleSubmit = async event => {
+    event.preventDefault()
+    try {
+      const response = await fetch(APIURL, {
+          method: "POST",
+          headers: { "Content-Type": "application/json" },
+          body: JSON.stringify({ message: newThought }),
+        })
+      
```

Comment:
Maybe there could be a way of adding the new new post into the list showing right away, instead of the user having to update the site to see the post?
Something like this: 
```
const json = await response.json()
setThoughts((previousThoughts) => [json, ...previousThoughts])